x/builder/rewards_address_provider/fixed_address_rewards_address_provider.go

```diff
@@ -0,0 +1,46 @@
+package rewardsaddressprovider
+
+import (
+	"cosmossdk.io/depinject"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/skip-mev/pob/x/builder/types"
+)
+
+// Provides auction profits to a fixed address
+type FixedAddressRewardsAddressProvider struct {
+	rewardsAddress sdk.AccAddress
+}
+
+func NewFixedAddressRewardsAddressProvider(
```

Comment:
Defaults for proposer rewards are zero, so they shld be stable for neutron. But I like this change, as sending anything to the zero address is likely an anti-pattern

Reply:
I think this is a non-actionable comment but please let me know if it isn't.